### PR TITLE
docs/440/add-discord-doc

### LINKED ISSUE #440: Add Channel Descriptions to Discord
**Description**

Addition of Channel Descriptions on Discord can be useful.
Suggested by @Vyvy-vi .

You can also join the discord community [here](https://discord.com/invite/jZQs6Wu)

Feel free to check out other cool repositories of Eddie Jaoude Community [here](https://github.com/EddieJaoudeCommunity)

### PR DESCRIPTION
**Pull request by nhcarrigan**
Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

closes #440 

#### What does this PR do?

Adds the content from the Discord #readme channel to a markdown file here for OS powered updates.

#### Description of Task to be completed?

None

#### How can this be manually tested?

View the file.

#### Any background context you want to provide?

See the discussion in the linked issue.

#### What is the relevant issue link?

Linked at the top.

#### Questions:

Are you having a fantastic day? 💜

### CODE REVIEW

File: discord-readme.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+-> #first-timers: get :green_square: to get you started in open source on GitHub - opportunities are provided for first-timers
+
+-> #content-share: share web content
+
+-> #virtual-events: events being hosted in the opensource community
+
+-> #bot-chat: discuss bots, esp. regarding development of EddieBot
+
+TECH: Channels regarding specific areas and languages
+->  #git, #cloud, #python, #linux, #javascript, #typescript, #devops, #tech-other, #php, #java, #go, #svelte, #c-langs.
```
Comment (BOLT04) [inline discord-readme.md:69]: if we add a new line i think all items are placed on the same line. What do you think?
```suggestion 

->  #git, #cloud, #python, #linux, #javascript, #typescript, #devops, #tech-other, #php, #java, #go, #svelte, #c-langs.
```

File: discord-readme.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+CHILLING: Channels to geek out in, mostly about off-topic stuff
+
+-> #moaning: for rant / venting and supporting each other
```
Comment (BOLT04) [inline discord-readme.md:73]: same comment here, what do you think of adding a new line between items so that each item has it's own line?

**Review by BOLT04 — COMMENTED** [review]
LGTM 👍 , just have a few suggestions @nhcarrigan to make each item on a new line. Currently it looks like this:
imagem

**Comment by nhcarrigan** [comment]
@BOLT04 - this should be corrected now. 🙂

**Review by BOLT04 — APPROVED** [review]
LGTM 👍, I can't merge it, but great job @nhcarrigan 😃

File: discord-readme.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+-> #chatting: geek out about off-topic content
+
+-> #healthfitness: health and fitness(not very active atm, lets improve this)
+
+-> #random: content that is off-topic and without relevance to topic
```
[the remark this answers is not in the document]
Comment (AllanRegush) [reply discord-readme.md:80]: Do we want this change?

**Review by AllanRegush — COMMENTED** [review]
Looks Good To Me :smile:

File: discord-readme.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (nhcarrigan) [reply discord-readme.md:80]: Sure 😁

[the remark this answers is not in the document]
Comment (nhcarrigan) [reply discord-readme.md:80]: Pinging @eddiejaoude to port this change over to the Discord readme channel.

**Comment by eddiejaoude** [comment]
@BOLT04 and @AllanRegush you both can approve and merge 🤓

**Comment by eddiejaoude** [comment]
Btw great team work 💪

**Review by AllanRegush — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**